src/components/Checkbox/Checkbox.spec.js

```diff
@@ -37,7 +37,7 @@ describe('Checkbox', () => {
     });
 
     it('should have mixed state when specified with the isIndeterminate prop', () => {
-        render(<Checkbox checked="mixed" aria-label="checkbox"/>);
+        render(<Checkbox defaultChecked="mixed" aria-label="checkbox"/>);
```

Comment:
I don't know why this change appears here, but defaultChecked is anyway a boolean and cannot take "mixed"

Reply:
You're right, maybe it was not the right place to fix that, but when I run the test I get this error:

> Warning: You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`.

So the solution will be to add an `onChange` function?

Reply:
I kept `checked=mixed` and added `onChange` to remove the warning